### DISCUSSION TITLE
Proposal - Research Paper Repo

### DISCUSSION THREAD

**Discussion by luisquintanilla** [fslaborg/FsLab#8] [Ideas]
I remember hearing during F# Conf that there are several published research papers that have used F#, specifically the FsLab set of libraries. 

It would be great to create a repository in the FsLab org to compile and aggregate these research papers. If code for these research papers is available, linking to that repository would be great as well. Think of it as the "Papers with Code" of FsLab.

If you're not familiar with "Papers with Code", here's the website. https://paperswithcode.com/ 

I'm not proposing having and maintaining a site like Papers With Code. A repo might be enough to showcase the papers and potentially their code.

As a next step, adding these papers to "Papers with Code" would be a good idea as well.

**Comment by luisquintanilla** [comment]
cc @kMutagene @muehlhaus

**Comment by kMutagene** [comment]
Great idea! Do you think it makes more sense to create a separate website/repo for this or would this be better as a subpage of the website?

I think it is nevertheless a great first step to just put up a place where people can add a link with description to their published F# research. Maybe there are such lists out there already?

**Comment by muehlhaus** [comment]
I really like this idea. I think it should be an additional page on fslab.org to promote the tool chain in terms of testimonials ....

**Comment by luisquintanilla** [comment]
@kMutagene @muehlhaus That's an even better idea. A subpage would be good enough for now and we can include instructions somewhere on the repo how to submit new papers. 

As far as lists of data science & ML publications that use F# / FsLab, I'm not sure. @dsyme are you aware of any? 

In the meantime, we can start with the ones you've published (if they're publicly available).

**Reply by dsyme** [reply]
There is https://fsharp.org/teaching/research.html but it's generally publications about F# or related CS topics, not publications by people who've happened to use F# along the way.
